data/strings/types_strings.txt

```diff
@@ -9514,6 +9517,7 @@
 
   [type.highway.footway.difficult_alpine_hiking]
     ref = type.highway.path
+    en = Hiking Trail
```

Comment:
It would be great to reflect the category/difficulty level of the track in the translation.

Reply:
Good point. I was a bit hesitant to add this because sac_scale terminology can't be translated literally when considering global usage. 

But I now tried to add some indication of difficulty that should be universally understandable.

Reply:
Why OSM terminology can't be used here directly?

Reply:
There was a discussion about it before in https://github.com/organicmaps/organicmaps/pull/2771#issuecomment-1164348210
(sorry, the most of it in russian, but the final correspondence between the tag and the en translation is there anyway)